src/main/java/com/benko/exercise/orchestra/Orchestra.java

```diff
@@ -0,0 +1,37 @@
+package com.benko.exercise.orchestra;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Orchestra implements com.benko.exercise.orchestra.Instrument {
+
+    private List<Instrument> instruments = new ArrayList<>();
+
+    private Instrument guitar = new Guitar();
+    private Instrument trombone = new Trombone();
+    private Instrument flute = new Flute();
+
+    private void addInstrument(){
+
+        instruments.add(guitar);
+        instruments.add(trombone);
+        instruments.add(flute);
+    }
+
+    public void play() {
+
+        addInstrument();
```

Comment:
You are adding instruments every time orchestra plays. That's not what a user of this class expects when invoking the `play` method.

Moreover consider example: 
```
orchestra.play();
orchestra.play();
```
The same orchestra on both cases but second time it will consist od 6 instruments


I propose: move adding instruments to the orchestra's constructor.